saleor/search/backends/postgresql_storefront.py

```diff
@@ -16,7 +16,7 @@ def search(phrase):
 
     """
     name_sim = TrigramSimilarity("name", phrase)
-    ft_in_description = Q(description__search=phrase)
+    ft_in_description = Q(search_vector=phrase)
```

Comment:
I would use the occasion and get rid of entire "saleor/search" module. We don't need it - it was used long time ago when we had ElasticSearch integration. If we wanted to add it now, we would implement it as a plugin. But for now it is not needed, so I would move this "search" function to where it is used: `graphql/product/filters.py`.